lib/api_catalogue.rb

```diff
@@ -1,6 +1,10 @@
+require "net/http"
 require_relative "api"
 require_relative "organisation"
 require_relative "csv_source"
+require_relative "fetcher/v1_alpha_fetcher"
+require_relative "registry_entry"
+require_relative "fetcher/fetcher_service"
```

Comment:
We should try and make it so we can just include `fetcher`, instead of each file we need:
```suggestion
require_relative "fetcher"
require_relative "registry_entry"
```

Reply:
as in move everything into one file? Or find a way to require every file in a directory?